libs/ui/src/components/RecordEdition/EditRecordContent/uiElements/StandardField/StandardFieldValue/DSRangePickerWrapper.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright LEAV Solutions 2017
+// This file is released under LGPL V3
+// License text available at https://www.gnu.org/licenses/lgpl-3.0.txt
+import {KitDatePicker} from 'aristid-ds';
+import {FunctionComponent, ReactNode} from 'react';
+import {IStandardFieldReducerState} from '../../../reducers/standardFieldReducer/standardFieldReducer';
+import {Form} from 'antd';
+import dayjs from 'dayjs';
+import {StandardValueTypes} from '../../../_types';
+import {styled} from 'styled-components';
+import {RangePickerProps} from 'antd/lib/date-picker';
+
+interface IDSRangePickerWrapperProps {
+    state: IStandardFieldReducerState;
+    infoButton: ReactNode;
+    value?: RangePickerProps['value'];
+    onChange?: RangePickerProps['onChange'];
+    handleSubmit: (value: StandardValueTypes, id?: string) => void;
+}
+
+const InputContainer = styled.div`
+    position: relative;
+
+    .actions {
+        position: absolute;
+        top: 53%;
+        right: 36px;
+        display: none;
+        z-index: 1000;
+    }
+
+    &:hover .actions {
+        display: block;
+    }
+`;
+
+export const DSRangePickerWrapper: FunctionComponent<IDSRangePickerWrapperProps> = ({
+    state,
+    infoButton,
+    value,
+    onChange,
+    handleSubmit
+}) => {
+    const {errors} = Form.Item.useStatus();
+    const isRequired = state.formElement.settings.required;
+
+    const _handleDateChange: (daysjsDates: [dayjs.Dayjs, dayjs.Dayjs], dateStrings: [string, string]) => void = (
+        daysjsDates,
+        dateStrings
+    ) => {
+        let dateToSave = {from: null, to: null};
+        if (daysjsDates) {
+            const [dateFrom, dateTo] = daysjsDates;
+            dateToSave = {from: String(dateFrom.unix()), to: String(dateTo.unix())};
+        }
+
+        onChange(daysjsDates, dateStrings);
+
+        if (isRequired && dateToSave.from === null) {
```

Comment:
**nitpick (blocking):** La condition n'est pas complète, elle n'est vrai que si :

* daysjsDates est vide ou non défini
* isRequired is true

Reply:
J'ai modifié pour tester la valeur en entrée `daysjsDates`.